src/main/java/de/epiceric/shopchest/config/Config.java

```diff
@@ -219,6 +219,11 @@
      **/
     public static boolean enableVendorMessages;
 
+    /**
+     * Whether the vendor of the shop should get messages about buys and sells
+     **/
+    public static boolean enableVendorBungeeMessages;
```

Comment:
add the property to the `config.yml` file

Reply:
I just added it to my commit: https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa